### PR TITLE
Fixed #981 and other minor fixes

### LINKED ISSUE #981: SaveAsPlugin does not save correctly the layers
The layers added to a map are not displayed when this one is loaded. The features property should be checked.

### PR DESCRIPTION
**Pull request by MV88**
Fixed #981
- Fixed the creation of the layers when the map is created. Was missing the features property.
- Fixed the behaviour of the modal in the Home. It wasn't resettiing the status after saving or closing the modal
- Fixed and added translations

### CODE REVIEW

File: web/client/translations/data.it-IT [hunk 1]
```diff
@@ -246,7 +249,7 @@
                         }
                     },
                     "polyline": {
-                        "error": "Errore: i bordi non si possono incrociare!",
```
Comment (offtherailz) [inline web/client/translations/data.it-IT]: You removed italian for english here

Comment (MV88) [inline web/client/translations/data.it-IT]: fixed

File: web/client/plugins/SaveAs.jsx [hunk 2]
```diff
@@ -157,24 +161,17 @@ module.exports = {
...
-
     })(assign(SaveAs, {
         BurgerMenu: {
             name: 'saveAs',
             position: 900,
             text: <Message msgId="saveAs"/>,
             icon: <Glyphicon glyph="floppy-open"/>,
-            action: editMap.bind(null, {}),
-            // display the BurgerMenu button only if the map can be edited
-            selector: (state) => {
```
Comment (Gnafu) [inline web/client/plugins/SaveAs.jsx]: Why the selector is removed?

**Comment by Gnafu** [comment]
I opened an existing map with an existing layer.
I added a new layer from the catalog and saved.
On reload:
the catalog layer is correctly displayed
the existing layer is not displayed

File: web/client/plugins/SaveAs.jsx (hunk 2 above)
Comment (MV88) [inline web/client/plugins/SaveAs.jsx]: Because Lorenzo Natali wanted to display the SaveAsPlugin in every map not only in the empty project

Comment (Gnafu) [inline web/client/plugins/SaveAs.jsx]: I will add the Selector again, otherwise anonymous users will have the Save As option that will not work.

**Pull request merged**